src/lib/components/Communities/CommunityHeader.js

```diff
@@ -45,30 +45,31 @@ class CommunityHeaderComponent extends Component {
               </>
             ) : (
               <div className="page-subheader-element">
-                {i18next.t('No community selected.')}
+                {i18next.t('Select the community where you want to upload your record.')}
```

Comment:
```suggestion
                {i18next.t('Select the community where you want to submit your record.')}
```
unless it was discussed and agreed in the mockups before, then please ignore this suggestion